File: common-web/src/main/webapp/js/service/usersService.js
//服务层
app.service('usersService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../users/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../users/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../users/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../users/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../users/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../users/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../users/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: common-web/src/main/java/com/pyg/manager/dao/UsersService.java
package com.pyg.manager.dao;
import java.util.List;

import com.pyg.manager.dao.pojo.Users;

import entity.PageResult;
/**
 * 服务层接口
 * 
 * @author Administrator
 *
 */
public interface UsersService {

	/**
	 * 返回全部列表
	 * 
	 * @return
	 */
	public List<Users> findAll();

	/**
	 * 返回分页列表
	 * 
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);

	/**
	 * 增加
	 */
	public void add(Users users);

	/**
	 * 修改
	 */
	public void update(Users users);

	/**
	 * 根据ID获取实体
	 * 
	 * @param id
	 * @return
	 */
	public Users findOne(String id);

	/**
	 * 批量删除
	 * 
	 * @param ids
	 */
	public void delete(String[] ids);

	/**
	 * 分页
	 * 
	 * @param pageNum
	 *            当前页 码
	 * @param pageSize
	 *            每页记录数
	 * @return
	 */
	public PageResult findPage(Users users, int pageNum, int pageSize);

}
